testing/log/service_test.go

```diff
@@ -167,7 +174,7 @@ func TestLogValidatorService_TestIndexedRecord(t *testing.T) {
 	context := manager.NewContext(toolbox.NewContext())
 	defer context.Close()
 	tempLog := path.Join(os.TempDir(), "endly_test_indexed.log")
-	toolbox.RemoveFileIfExist(tempLog)
+	_ = toolbox.RemoveFileIfExist(tempLog)
```

Comment:
what is toolbox.RemoveFileIfExist(fullLogname) return? why you use blank identifier here ?

Reply:
as above